pullRequestHandler.go

```diff
@@ -133,3 +142,10 @@ func hasUnsigned(req types.PullRequestOuter, client *github.Client) (bool, error
 func isSigned(msg string) bool {
 	return strings.Contains(msg, "Signed-off-by:")
 }
+
+func firstTimeContributor(label string) bool {
```

Comment:
If you prefix this with `func (p *PullRequest)` then the method will be attached to the PullRequest object itself which may be clear than having a function defined in the package.

Reply:
This change could be done in a follow-up PR.

Reply:
I cannot make the function method since the struct is in types.go and my function is in pullRequestHandler.I can move the function in types.go but i think this will ruin the point of the files which i think is to be filled with structs only?

Reply:
You don't need to move the code, Go can figure it out